frontend/www/js/omegaup/components/problem/FilterTags.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="card">
+    <div class="card-body">
+      <h3>{{ T.problemEditAddTags }}</h3>
+      <div v-for="(tag, index) in tags" :key="index" class="form-check">
+        <input
+          :id="tag.tagname"
+          v-model="localValue"
+          class="form-check-input"
+          type="checkbox"
+        />
+        <label class="form-check-label" :for="tag">{{ T[tag] }}</label>
```

Comment:
Aquí hay algunas cosas que se pueden arreglar:

1.- Tratamos de evitar en lo mayor posible el uso del atributo `id` en vue, debido a que estos pudieran colisionar con id's de otros componentes que se muestran en la misma vista.
2.- Si el motivo de usar el id es para ligarlo al label y que se seleccione el check cuando presionas el texto, gracias a vue, conocimos un pequeño truco que nos ayuda a obtener el mismo resultado: Envuelves el input y el texto dentro de la etiqueta `label`.
3.- Faltaba que incluyeras la directiva `v-bind:value` o su equivalente simplificado `:value` el cual debe contener el valor del tag.

Mas o menos podría quedar así:
```suggestion
        <label class="form-check-label">
          <input
            :value="tag"
            v-model="localValue"
            class="form-check-input"
            type="checkbox"
          />{{ T[tag] }}
        </label>
```